exporter/metric/metric_test.go

```diff
@@ -28,8 +29,12 @@ import (
 	"go.opentelemetry.io/otel/sdk/metric/aggregator/lastvalue"
 	aggtest "go.opentelemetry.io/otel/sdk/metric/aggregator/test"
 	"go.opentelemetry.io/otel/sdk/resource"
+	"google.golang.org/api/option"
 
 	googlemetricpb "google.golang.org/genproto/googleapis/api/metric"
+
```

Comment:
The import sorting/grouping seems random, please fix

Reply:
Done